### PR TITLE
Added `approximate` number field

### PR DESCRIPTION
**Pull request by tudny**
When doing numerical methods calculations, approximate values are way more useful.

### CODE REVIEW

**Comment by tudny** [comment]
Testing evidence
image

**Comment by tudny** [comment]
@mgr0dzicki plz review

File: src/float.rs [hunk 1]
```diff
@@ -0,0 +1,310 @@
...
+/// # Returns
+/// A string representing the same floating point number, but without trailing zeros
+/// # Examples
+/// ```rust
+/// # use crate::jp2gmd_lib::trim_trailing_zeros_float_str;
+/// assert_eq!(trim_trailing_zeros_float_str("10.0"), "10");
+/// assert_eq!(trim_trailing_zeros_float_str("123.450"), "123.45");
+/// assert_eq!(trim_trailing_zeros_float_str("1.0"), "1");
+/// ```
+pub fn trim_trailing_zeros_float_str(s: &str) -> String {
```
Comment (tudny) [inline src/float.rs:225]: `s` - very descriptive name @tudny

**Review by mgr0dzicki — APPROVED** [review]
Ok, convinced, nice! 

I regret we cannot type floats instead of rationals, but I guess that command line parsing is on me... Will do someday!

File: src/float.rs (hunk 1 above)
Comment (mgr0dzicki) [reply src/float.rs:225]: does it really matter if this is an only argument and function name is already descriptive?

Comment (tudny) [reply src/float.rs:225]: It does, as the function doesn't only remove trailing zeros from the given string, but trims the zeros from the float number represented by this string. E.g. 
10.0 -> 10 (the dot is removed as well)
10 -> 10 (there is a trailing zero, that is not removed)

**Comment by tudny** [comment]
> Ok, convinced, nice! 
> 
> I regret we cannot type floats instead of rationals, but I guess that command line parsing is on me... Will do someday!

You can add an Issue!

**Pull request merged**